Makefile

```diff
@@ -143,7 +143,7 @@ pypi:  ## Release package to pypi
 	@pyenv rehash > /dev/null 2> /dev/null; true
 
 .deps/flake8: | .deps/flake8_mock .deps/flake8_tuple .deps/flake8_string_format .deps/flake8_debugger .deps/flake8_deprecated .deps/flake8_comprehensions
-	pip install --upgrade 'flake8<3.0.0'
+	pip install --upgrade 'flake8<4.0.0'
```

Comment:
You can remove the <3.0.0 completely. It was a workaround that is not needed anymore.